src/localization/formInputStrings.json

```diff
@@ -27,7 +27,8 @@
     "policy_type": "Policy type",
     "registration_code": "Registration code",
     "unique_personal_policy": "Must be a unique personal policy number between 0 and 50 characters.",
-    "yes": "yes"
+    "yes": "yes",
```

Comment:
🤔 should your "yes" be "Yes"?
The "Oui" is capitalised.

Reply:
c05de31